### PR TITLE
update FIX workshop 

### PR DESCRIPTION
**Pull request by jc-cb**
see slack for change log

### CODE REVIEW

File: FIX/app/configuration.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Coinbase Global, Inc.
+# Copyright 2023 Coinbase Global, Inc.
```
Comment (nmorgancb) [inline FIX/app/configuration.py:1]: 2023-present

File: FIX/app/fix_session.py [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+    def onLogon(self, sessionID):
+        """Function called upon Logon"""
+        logfix.info('---------------Successful Logon----------------')
+        self.sessionID = sessionID
+        return
+
+    def onLogout(self, sessionID):
+        """Function called upon Logout"""
+        #logfix.info('Session (%s) logout!' % sessionID.toString())
```
Comment (rcbgr) [inline FIX/app/fix_session.py:115]: Remove this commented out string?

File: FIX/app/fix_session.py [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+    def logout(self, fixSession, sessionID):
+        """Build Cancel Order Message (F) based-on user input"""
+        logout_message = fix.Message()
+        header = logout_message.getHeader()
+        header.setField(fix.MsgType(fix.MsgType_Logout))
+        fixSession.send_message(logout_message)
+        time.sleep(1)
+        sys.exit()
+
```
Comment (rcbgr) [inline FIX/app/fix_session.py:220]: Remove extra line

File: LICENSE [hunk 4]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 Coinbase Global, Inc.
+   Copyright 2023 Coinbase Global, Inc.
```
Comment (rcbgr) [inline LICENSE:189]: 2023-present

**Review by rcbgr — CHANGES_REQUESTED** [review]

**Review by jprix — APPROVED** [review]

**Review by rcbgr — APPROVED** [review]

**Review by KMORGAN008 — APPROVED** [review]

**Comment by KMORGAN008** [comment]
e8f76d65c46b580e52f5bbea5f5a31781fd8b1a0

**Pull request merged**